azurelinuxagent/common/osutil/default.py

```diff
@@ -109,6 +113,10 @@ def _get_firewall_delete_conntrack_drop_command(wait, destination):
                      ["iptables", "-t", "security", "-D", "OUTPUT", "-d", destination, "-p", "tcp", "-m", "conntrack",
                       "--ctstate", "INVALID,NEW", "-j", "DROP"])
 
+def _get_firewall_delete_accept_nonroot_tcp_command(wait, destination):
+    return _add_wait(wait,
```

Comment:
Can this be parameterized similar to `AddFirewallRules.get_iptables_accept_command_nonroot_tcp(wait, command, destination)`

Reply:
I have written it the way other delete rules have been coded . Do you think all of them needs to be parameterized ?

Reply:
Addressed